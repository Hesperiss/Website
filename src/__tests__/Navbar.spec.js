import React from "react";
import NavBarLanding from '../Components/Landing/Components/Navbar';

describe("Navbar", () => {
  test("Matches the snapshot", () => {
    const navbar = <NavBarLanding/>;
    expect(navbar).toMatchSnapshot();
  });
});