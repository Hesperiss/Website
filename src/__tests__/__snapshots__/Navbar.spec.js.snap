// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Navbar Matches the snapshot 1`] = `<NavBarLanding />`;
